proto/rrg/winreg.proto

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023 Google LLC
```

Comment:
Should it be Copyright **2024**?

Reply:
Ah, right. Fixed here and in `get_winreg_value.proto` and in `get_winreg_value.rs`.